backend-lucene/src/main/java/org/hibernate/search/v6poc/backend/lucene/document/model/impl/IndexSchemaFieldLocalDateContext.java

```diff
@@ -169,16 +156,19 @@ public int hashCode() {
 		}
 
 		private int buildHashCode() {
-			return Objects.hashCode( store );
+			return Objects.hash( store, sortable );
 		}
 	}
 
 	private static final class LocalDateFieldQueryFactory implements LuceneFieldQueryFactory {
 
 		private final LocalDateFieldFormatter localDateFieldFormatter;
 
+		private final int hashCode;
+
 		private LocalDateFieldQueryFactory(LocalDateFieldFormatter localDateFieldFormatter) {
 			this.localDateFieldFormatter = localDateFieldFormatter;
+			this.hashCode = buildHashCode();
```

Comment:
From what I can see we don't even use the hashcode for this type (even indirectly through `LuceneIndexSchemaFieldNode`), so this looks like very premature optimisation :) But as you wish.

Reply:
Yeah, it's an habit coming from HV but let's keep it for the elements appearing in a Map. Removed.